src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -61,6 +57,14 @@ function AxisSystem(props: Props): JSX.Element {
     ),
   };
 
+  const xAxisScale = axisToCanvas.x.copy();
+  xAxisScale.domain(visibleDomains.bottom);
+  xAxisScale.range([0, width]);
+
+  const yAxisScale = axisToCanvas.y.copy();
+  yAxisScale.domain(visibleDomains.left);
+  yAxisScale.range([height, 0]);
```

Comment:
Visible axis scales are copied from axisToCanvas scales so that it uses the appropriate linear or log scale.